packages/react-components/package.json

```diff
@@ -71,6 +73,8 @@
     "vitest": "0.34.1"
   },
   "peerDependencies": {
+    "@agoric/rpc": "^0.9.0",
+    "@agoric/web-components": "^0.15.0",
```

Comment:
They're either `dependencies` or `peerDependencies` but not both.

peer has less burden on the consumer, so if that works go with it

Reply:
I think they should be `dependencies` because they're imported and used directly in this package: https://github.com/Agoric/ui-kit/blob/main/packages/react-components/src/lib/context/AgoricProviderLite.tsx#L10-L18

It's possible the consumer doesn't directly use the exposed `chainStorageWatcher` or `walletConnection` types from the context so I think it would be more burden to require them to add these as dependencies on their own.

However, if the client is using those types elsewhere, I figured it would make sense to put them in `peerDependencies` as well to avoid conflicts. Am I thinking about this wrong?

Reply:
yeah then they're `dependencies`.

I see your motivation to constrain the `peerDependencies` as well, but I don't think it's warranted.